### PR TITLE
Test html line prefixes

### PR DESCRIPTION
**Pull request by rtfpessoa**

### CODE REVIEW

**Comment by rtfpessoa** [comment]
@pbu88 just did some testing with an html parser to avoid white space problems, which allows element searching. What do you think?

File: test/side-by-side-printer-tests.js [hunk 1]
```diff
@@ -21,4 +21,99 @@ describe('SideBySidePrinter', function() {
...
   });
+
+  describe('generateSideBySideFileHtml', function() {
+    it('should generate lines with the right prefixes', function() {
+      var sideBySidePrinter = new SideBySidePrinter({});
+
+      var file = {
+        "blocks": [{
+          "lines": [{
+            "content": " context",
```
Comment (rtfpessoa) [inline test/side-by-side-printer-tests.js]: Codacy Issue found: [Strings must use singlequote.](https://www.codacy.com/app/Codacy/diff2html/file/1654079114/issues/source?bid=3025341&fileBranchId=3037234#l32)

File: test/side-by-side-printer-tests.js [hunk 2]
```diff
@@ -21,4 +21,99 @@ describe('SideBySidePrinter', function() {
...
+            "type": "d2h-ins",
+            "oldNumber": null,
+            "newNumber": 20
+          }], "oldStartLine": "19", "newStartLine": "19", "header": "@@ -19,7 +19,7 @@"
+        }],
+        "deletedLines": 1,
+        "addedLines": 1,
+        "checksumBefore": "fc56817",
+        "checksumAfter": "e8e7e49",
+        "mode": "100644",
```
Comment (rtfpessoa) [inline test/side-by-side-printer-tests.js]: Codacy Issue found: [Strings must use singlequote.](https://www.codacy.com/app/Codacy/diff2html/file/1654079114/issues/source?bid=3025341&fileBranchId=3037234#l47)

File: test/side-by-side-printer-tests.js [hunk 3]
```diff
@@ -21,4 +21,99 @@ describe('SideBySidePrinter', function() {
...
+
+  describe('generateSideBySideFileHtml', function() {
+    it('should generate lines with the right prefixes', function() {
+      var sideBySidePrinter = new SideBySidePrinter({});
+
+      var file = {
+        "blocks": [{
+          "lines": [{
+            "content": " context",
+            "type": "d2h-cntx",
```
Comment (rtfpessoa) [inline test/side-by-side-printer-tests.js]: Codacy Issue found: [Strings must use singlequote.](https://www.codacy.com/app/Codacy/diff2html/file/1654079114/issues/source?bid=3025341&fileBranchId=3037234#l33)

File: test/side-by-side-printer-tests.js [hunk 4]
```diff
@@ -21,4 +21,99 @@ describe('SideBySidePrinter', function() {
...
+        }],
+        "deletedLines": 1,
+        "addedLines": 1,
+        "checksumBefore": "fc56817",
+        "checksumAfter": "e8e7e49",
+        "mode": "100644",
+        "oldName": "coverage.init",
+        "language": "init",
+        "newName": "coverage.init",
+        "isCombined": false
```
Comment (rtfpessoa) [inline test/side-by-side-printer-tests.js]: Codacy Issue found: [Strings must use singlequote.](https://www.codacy.com/app/Codacy/diff2html/file/1654079114/issues/source?bid=3025341&fileBranchId=3037234#l51)

File: test/side-by-side-printer-tests.js [hunk 5]
```diff
@@ -21,4 +21,99 @@ describe('SideBySidePrinter', function() {
...
+          "lines": [{
+            "content": " context",
+            "type": "d2h-cntx",
+            "oldNumber": 19,
+            "newNumber": 19
+          }, {"content": "-removed", "type": "d2h-del", "oldNumber": 20, "newNumber": null}, {
+            "content": "+added",
+            "type": "d2h-ins",
+            "oldNumber": null,
+            "newNumber": 20
```
Comment (rtfpessoa) [inline test/side-by-side-printer-tests.js]: Codacy Issue found: [Strings must use singlequote.](https://www.codacy.com/app/Codacy/diff2html/file/1654079114/issues/source?bid=3025341&fileBranchId=3037234#l40)

File: test/side-by-side-printer-tests.js [hunk 6]
```diff
@@ -21,4 +21,99 @@ describe('SideBySidePrinter', function() {
...
+          }], "oldStartLine": "19", "newStartLine": "19", "header": "@@ -19,7 +19,7 @@"
+        }],
+        "deletedLines": 1,
+        "addedLines": 1,
+        "checksumBefore": "fc56817",
+        "checksumAfter": "e8e7e49",
+        "mode": "100644",
+        "oldName": "coverage.init",
+        "language": "init",
```
Comment (rtfpessoa) [inline test/side-by-side-printer-tests.js]: Codacy Issue found: [Strings must use singlequote.](https://www.codacy.com/app/Codacy/diff2html/file/1654079114/issues/source?bid=3025341&fileBranchId=3037234#l49)

File: test/side-by-side-printer-tests.js [hunk 7]
```diff
@@ -21,4 +21,99 @@ describe('SideBySidePrinter', function() {
...
+  describe('generateSideBySideFileHtml', function() {
+    it('should generate lines with the right prefixes', function() {
+      var sideBySidePrinter = new SideBySidePrinter({});
+
+      var file = {
+        "blocks": [{
+          "lines": [{
+            "content": " context",
+            "type": "d2h-cntx",
+            "oldNumber": 19,
```
Comment (rtfpessoa) [inline test/side-by-side-printer-tests.js]: Codacy Issue found: [Strings must use singlequote.](https://www.codacy.com/app/Codacy/diff2html/file/1654079114/issues/source?bid=3025341&fileBranchId=3037234#l34)

File: test/side-by-side-printer-tests.js [hunk 8]
```diff
@@ -21,4 +21,99 @@ describe('SideBySidePrinter', function() {
...
+            "newNumber": 19
+          }, {"content": "-removed", "type": "d2h-del", "oldNumber": 20, "newNumber": null}, {
+            "content": "+added",
+            "type": "d2h-ins",
+            "oldNumber": null,
+            "newNumber": 20
+          }], "oldStartLine": "19", "newStartLine": "19", "header": "@@ -19,7 +19,7 @@"
```
Comment (rtfpessoa) [inline test/side-by-side-printer-tests.js]: Codacy Issue found: [Strings must use singlequote.](https://www.codacy.com/app/Codacy/diff2html/file/1654079114/issues/source?bid=3025341&fileBranchId=3037234#l41)

File: test/side-by-side-printer-tests.js [hunk 9]
```diff
@@ -21,4 +21,99 @@ describe('SideBySidePrinter', function() {
       assert.equal(expectedLeft, fileHtml.left);
     });
   });
+
+  describe('generateSideBySideFileHtml', function() {
+    it('should generate lines with the right prefixes', function() {
+      var sideBySidePrinter = new SideBySidePrinter({});
+
+      var file = {
+        "blocks": [{
```
Comment (rtfpessoa) [inline test/side-by-side-printer-tests.js]: Codacy Issue found: [Strings must use singlequote.](https://www.codacy.com/app/Codacy/diff2html/file/1654079114/issues/source?bid=3025341&fileBranchId=3037234#l30)

File: test/side-by-side-printer-tests.js [hunk 10]
```diff
@@ -21,4 +21,99 @@ describe('SideBySidePrinter', function() {
...
+            "oldNumber": null,
+            "newNumber": 20
+          }], "oldStartLine": "19", "newStartLine": "19", "header": "@@ -19,7 +19,7 @@"
+        }],
+        "deletedLines": 1,
+        "addedLines": 1,
+        "checksumBefore": "fc56817",
+        "checksumAfter": "e8e7e49",
+        "mode": "100644",
+        "oldName": "coverage.init",
```
Comment (rtfpessoa) [inline test/side-by-side-printer-tests.js]: Codacy Issue found: [Strings must use singlequote.](https://www.codacy.com/app/Codacy/diff2html/file/1654079114/issues/source?bid=3025341&fileBranchId=3037234#l48)

**Comment by pbu88** [comment]
Yeah I think is great! :+1:

**Pull request merged**